PetaPoco/Database.cs

```diff
@@ -3365,7 +3373,9 @@ public string FormatCommand(string sql, object[] args)
                 sb.Append("\n");
                 for (int i = 0; i < args.Length; i++)
                 {
-                    sb.AppendFormat("\t -> {0}{1} [{2}] = \"{3}\"\n", _paramPrefix, i, args[i].GetType().Name, args[i]);
+                    var arg = args[i];
+                    var argTypeName = arg == null ? "Unknown Type" : args.GetType().Name;
+                    sb.AppendFormat("\t -> {0}{1} [{2}] = \"{3}\"\n", _paramPrefix, i, argTypeName, arg);
```

Comment:
This should have the same result, while remaining as concise as possible:

```suggestion
                    sb.AppendFormat("\t -> {0}{1} [{2}] = \"{3}\"\n", _paramPrefix, i, args[i]?.GetType().Name ?? "Unknown Type", args[i]);
```

Reply:
I didn't want to do `args[i]` twice and didn't see any other code using the _null conditional_ operator. I remember trying it somewhere and it gave me a syntax error...Unless I'm thinking about `var a ??= b;` or `var a = b ?? c;`
But it's actually only an index lookup i.e. no performance overhead. Agreed.

Reply:
Correct, null conditional with assignment isn't supported...net450 and their partial support for null handling definitely feels very WIP lol.